### PR TITLE
Add re-calculation of pool liquidity using the SDK

### PR DESCRIPTION
**Pull request by timjrobinson**
Adding function to take a list of pools + token prices which then uses the SDK to re-calculate the liquidity of each pool and return the updated pool. This will then be used to update all pools in DynamoDB and then in a future PR I'll add a lambda that runs this function each minute so pool liquidity is always kept up to date.

### CODE REVIEW

**Comment by timjrobinson** [comment]
Requires [#104](https://github.com/balancer-labs/balancer-sdk/pull/104) to be merged before this will build.

**Comment by timjrobinson** [comment]
I did a major refactor of the price system in the pools API. previously the price was just a string that represented the native asset price in the token. It has now been changed to being a dictionary of currency -> price like the frontend/sdk holds prices (e.g. `{usd: '5', eth: '0.001', 'matic': '2'}`). 

This makes it much easier to calculate liquidity, as it can just take the usd prices from the database instead of having to re-calculate them for each network. 

SOR appears to be working correctly with the updates, I compared common queries with this code vs what SOR returns in prod and they are the same.

**Comment by timjrobinson** [comment]
Fixed up all the tests, added new ones for SOR so that it can handle prices as a string or an object (for backwards compatibility so SOR works while prices are being updated to the new style). Also made updatePools save the total liquidity as `{graphData: {totalLiquidity}}` so that it doesn't overwrite the calculated data and we can continually compare calculated liquidity to what the graph gives to look for discrepencies.

File: lambdas/decorate-pools.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+/**
+ * This lambda runs after prices have been updated, it updates the liquidity of each pool
+ * based on the latest token prices + pool token values.
```
[the remark this answers is not in the document]
Comment (timjrobinson) [reply lambdas/decorate-pools.ts:4]: At the moment it actually runs every minute. Could make it also run after pools are updated.

**Review by garethfuller — APPROVED** [review]

**Pull request merged**